include/oc_export.h

```diff
@@ -0,0 +1,42 @@
+/*
+// Copyright (c) 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#ifndef OC_EXPORT_H
+#define OC_EXPORT_H
+
+#if defined _WIN32 || defined __CYGWIN__ || defined __MINGW32__
```

Comment:
I'm not sure whether `__CYGWIN__` and `__MINGW32__` use the `__declspec()` keyword. If I remember correctly, they also use use the `__attribute__` keyword. Is that correct?

Reply:
Seems that for `__CYGWIN__` and `__MINGW32__` _both_ ways work. So I can simplify the expression and remove the `__CYGWIN__` and `__MINGW32__` subconditions. (Similar as it is in https://github.com/zeromq/libzmq/blob/master/include/zmq.h#L62 .)

Reply:
> Seems that for `__CYGWIN__` and `__MINGW32__` _both_ ways work.

Ah, interesting - I didn't know that.